src/styles/blocks/calendar.scss

```
@@ -0,0 +1,50 @@
+.calendar {
+  position: relative;
+  top: 50%;
+  margin: auto;
+  transform: translateY(25%);
+  width: (7 * $cell-size) + 7*$gap;
+  
+  &__field {
+    display: flex;
+    flex-wrap: wrap;
+    gap: $gap;
+  
+    @each $day, $value in $days {
+      &--start-day--#{$day} :first-child {
+        margin-left: ($gap + $cell-size) * ($value - 1);
+      }
+    }
+    
+    @for $day from 29 through 31 {
+      &--month-length--28 :nth-child(n + #{$day + 1}) {
```

there should be a dynamic value to make it work, as Alisa suggested:
```suggestion
      &--month-length--#{$day} :nth-child(n + #{$day + 1}) {
```

---

I put #{$day} instead of 28 in line20 and nothing has changed. calendar`s last days still are visiable.

---

And about this part. 
my bad, I had to write * selector instead of html
it should be applied to all elements on the page